assets/scss/variables.scss

```diff
@@ -42,11 +42,21 @@ $defaultTagColors: #fff, #fff, #fff, #fff, #fff;
 :root {
     --sys-font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Droid Sans", "Helvetica Neue";
     --zh-font-family: "PingFang SC", "Hiragino Sans GB", "Droid Sans Fallback", "Microsoft YaHei";
+    --cjk-font-family: "Source Han Sans", "Noto Sans CJK";
+    --ja-font-family: "Hiragino Kaku Gothic ProN", "Meiryo", "Noto Sans CJK JP";
 
-    --base-font-family: "Lato", var(--sys-font-family), var(--zh-font-family), sans-serif;
+    --base-font-family: "Lato", var(--sys-font-family), var(--cjk-font-family), sans-serif;
     --code-font-family: Menlo, Monaco, Consolas, "Courier New", monospace;
 }
 
+body:lang(ja) {
+    --base-font-family: var(--ja-font-family), var(--sys-font-family), var(--cjk-font-family), sans-serif;
+}
+
+body:lang(zh) {
+    --base-font-family: var(--zh-font-family), var(--sys-font-family), var(--cjk-font-family), sans-serif;
```

Comment:
I'm not really sure about this value.

I would write:

```
--base-font-family: "Lato", var(--sys-font-family), var(--zh-font-family),  var(--cjk-font-family), sans-serif;
```

So:

1. **Lato is used for the Latin alphabet**
2. In case Lato is not available, use `--sys-font-family` (which is optimized for the Latin alphabet)
3. For CJK text, it won't match any of the previous cases, so it will try to use `--zh-font-family` and succeed.

The same rule applies to `ja`. It's important to have "Lato" in the first place, otherwise the Latin alphabet would use the CJK font and look really bad.

Does that make sense to you?

Reply:
Thank you for your review, and I apologize for the delayed response.
Your comments were correct, so I have made the necessary revisions. You might notice several push --force operations, which occurred due to my unfamiliarity with GitHub. Please note that these do not affect the content of the commits. Thank you for your understanding.